lib/datasets/rdatasets.rb

```diff
@@ -0,0 +1,126 @@
+require_relative "dataset"
+require_relative "tar_gz_readable"
+
+module Datasets
+  class RDatasets < Dataset
+    class FileNotFound < StandardError; end
+
+    class Master < Dataset
```

Comment:
Could your move this to `Datasets::RDatasetsList` like existing `Datasets::LIBSVMDatasetList`?

Reply:
I see.